python/setup.py

```diff
@@ -52,6 +52,13 @@ def get_cuda_version_from_header(cuda_include_dir):
 cuda_lib_dir = os.path.join(CUDA_HOME, "lib64")
 CUDA_VERSION = get_cuda_version_from_header(cuda_include_dir)
 
+INSTALL_PREFIX = os.environ.get("INSTALL_PREFIX", False)
+if os.path.isdir(INSTALL_PREFIX):
+    rmm_include_dir = os.path.join(INSTALL_PREFIX, "include")
+else:
+    # use uninstalled headers in source tree
+    rmm_include_dir = "../include"
```

Comment:
Can we include `../build/include` in here as well just to be safe?

Reply:
I can if you really want me to, but this directory doesn't even get created anymore. I believe it was used for `cnmem.h`, which is gone.

Reply:
Apologies, I thought that they were copied into a `../build/include` directory for some reason. This should be good to go, thanks!